src/index.ts

```diff
@@ -165,6 +164,11 @@ export abstract class DetSysAction {
 
   constructor(actionOptions: ActionOptions) {
     this.actionOptions = makeOptionsConfident(actionOptions);
+    this.idsHost = new IdsHost(
+      this.actionOptions.idsProjectName,
+      actionOptions.diagnosticsSuffix,
+      process.env["INPUT_DIAGNOSTIC-ENDPOINT"],
```

Comment:
Is it necessary to use an env var here rather than `inputs.getString`?

Reply:
Unfortunately yes, just added this comment:

```
    // Note: we don't use actionsCore.getInput('diagnostic-endpoint') on purpose:
    // getInput silently converts absent data to an empty string.
```